src/dev/build/args.ts

```diff
@@ -111,6 +114,9 @@ export function readCliArgs(argv: string[]) {
     createDebPackage: isOsPackageDesired('deb'),
     createDockerPackage: isOsPackageDesired('docker'),
     createDockerUbiPackage: isOsPackageDesired('docker') && !Boolean(flags['skip-docker-ubi']),
+    darwinX64: Boolean(flags['darwin-x64']),
```

Comment:
should this be targetDarwinX64? or something similar.

Reply:
replaced with interface object targetPlatforms